spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -31,47 +31,117 @@ object SpdxLicenseAliasMapping {
      */
     private val customNames = mapOf(
         "AFLv2.1" to AFL_2_1,
+        "afl" to AFL_2_1,
```

Comment:
@tsteenbe, why is this mapped to `AFL_2_1` and not to latest version `AFL_3_0`? For other mappings we also seem to assume the latest version if no explicit version is given.